### ISSUE TITLE
Minimap gone

### ISSUE THREAD

**Issue by Ashzure** [nmsmith22389/CandyUI#91]
Everything works fine except for the minimap is not showing. My guess is that its outside the screen boundary somewhere and resetting it dosen't move back.

**Comment by coding-red-panda** [comment]
try using the windows options from the client:

escape -> interface -> positioning

And see if you can reset the window's position there.

**Comment by coding-red-panda** [comment]
If that fails, delete the file called CandyUI_Minimap_0_Acct.xml, localed under %AppData%\NCsoft\Wildstar\AddonSaveData\<account folder>

**Comment by Ashzure** [comment]
Shouldve said i tried both of those things before posting

**Comment by coding-red-panda** [comment]
...then I really have no idea :/
I can try building in a reset command to put it back on the correct position.

Can you open the interface menu as described above and tell me the coordinates it's supposed to be located at?

**Comment by Ashzure** [comment]
2575x35 on 1440p monitor

**Comment by coding-red-panda** [comment]
yep that's out of bounds if I'm not mistaken, as your resolution should stop at 2560.

Wild idea : try scaling down the UI to a different resolution and see if it shows up?

**Comment by coding-red-panda** [comment]
What you could try is the following in chat:

```
/eval Apollo.GetAddon("CandyUI_Minimap").wndMain:Move(1750, 35, 100 ,100)
```

This should hopefully move it more or less to the center of your screen.
This is from the top of my head, so might throw an error.

**Comment by Ashzure** [comment]
That worked

**Comment by coding-red-panda** [comment]
okay, keeping that as reference open for now then :)
And will try to build something in CandyUI to reset the position of windows to something like `0 0`  so it appears on screen again

**Comment by lucasvienna** [comment]
I will second this bug. Sadly the CUI control panel is sending it off the map when resizing

**Comment by coding-red-panda** [comment]
thanks for the feedback, should help me pinpoint as to where to find the issue.
